cmd/dep/status.go

```diff
@@ -369,12 +369,10 @@ func runStatusAll(out outputter, p *dep.Project, sm *gps.SourceMgr) error {
 	rm, _ := ptree.ToReachMap(true, true, false, nil)
 
 	external := rm.Flatten(false)
-	roots := make(map[gps.ProjectRoot][]string)
-	var errs []string
+	roots := make(map[gps.ProjectRoot][]string, len(external))
 	for _, e := range external {
 		root, err := sm.DeduceProjectRoot(e)
 		if err != nil {
-			errs = append(errs, string(root))
 			continue
 		}
```

Comment:
@y0ssar1an ah sorry, I noticed this over in another PR and pushed a temporary fix directly. (This is actually a real problem, but fixing it requires going back to the drawing board a bit on how status works, as it entails a new output mode)